### ISSUE TITLE
Pick which themes to switch between

### ISSUE THREAD

**Issue by lachlanjc** [jamox/hyperambient#2]
I love the idea of hyperambient, but I’d love to be able to customize which themes it switches between. Is that possible?
Thanks 😀

**Comment by jamo** [comment]
It's the first [TODO item](https://github.com/jamox/hyperambient#todo) of the Readme. 

I hope the feature will be available shortly...

**Comment by jamo** [comment]
It's now implemented (https://github.com/jamox/hyperambient/commit/21da1ee201177ae1562287fb7585b33bebe494cf) and included in the latest release (0.3.0) 

You'll need to create a `~/.hyperambient.json` config like this:

``` javascript
{
  "config": {
    "darkTheme": "hyperterm-gruvbox-dark",
    "lightTheme": "hyperterm-one-light"
  }
}
```

and run the you'l need to run `Plugins -> Update all now` at least once, maybe twice
Full explantion in readme: https://github.com/jamox/hyperambient#customizing

**Comment by lachlanjc** [comment]
Fabulous! Thanks 👏🏻
